py/torch_tensorrt/dynamo/lowering/_decompositions.py

```diff
@@ -412,7 +412,8 @@ def get_decompositions(
         return {**CORE_ATEN_DECOMPOSITIONS_FILTERED, **TORCH_TRT_DECOMPOSITIONS}
     else:
         # changes made here due to torch2.6 changes https://github.com/pytorch/pytorch/pull/135080
-        decomp_table = _decomp_table_to_post_autograd_aten()
```

Comment:
Is it possible to remove this change from this PR since its in another or we can do a stacked merge

Reply:
I stacked the PR as suggested.